apps/admin/frontend/src/screens/unconfigured_screen.tsx

```diff
@@ -61,8 +64,25 @@ const demoElection =
 export function UnconfiguredScreen(): JSX.Element {
   const history = useHistory();
   const isMounted = useMountedState();
+  const configureMutation = configure.useMutation();
+
+  const configureMutateAsync = useCallback(
+    async (electionData) => {
+      return new Promise<ConfigureResult>((resolve) => {
+        configureMutation.mutate(
+          {
+            electionData,
+          },
+          {
+            onSuccess: resolve,
+          }
+        );
+      });
+    },
+    [configureMutation]
+  );
```

Comment:
This is basically the same as `configureMutation.mutateAsync` except it doesn't throw errors. I wonder if it would be clearer to just use `mutateAsync` here and catch the errors. Reading this code, it's a little unclear why we're using `mutate`

Reply:
I misunderstood your previous comments and I thought I had to always use `mutate` or else the errors would escape off into the abyss. I'm going to need an explainer on this.

Reply:
So, _normally_ the unhandled promise would crash the app but you've added code _across apps_ so that's not the case?

In `jest`, wouldn't the rejected promise only happen if you mocked the rejection? So you'd know where it is?

If the confusion and crashing is only in test environments, and I don't expect this mutation to throw errors, then I don't see why I wouldn't just use `await configure.mutateAsync()`?

Reply:
> So, normally the unhandled promise would crash the app but you've added code across apps so that's not the case?

Yep!

> In jest, wouldn't the rejected promise only happen if you mocked the rejection? So you'd know where it is?

The main way it happens is if the API mock calls are incorrect and there's a missing mock call, which immediately throws an error. Even if you don't throw an error and just log an error (which is an option I built in the API mock), you still need to return some dummy value, which may trigger an error since the consuming code might be expecting a different shape. All of these cases lead to difficult to debug jest crashes.